### ISSUE TITLE
Build issue

### ISSUE THREAD

**Issue by selvam920** [bitsdojo/bitsdojo_window#125]
i'm getting build issue in master branch

Flutter 2.10.0-1.0.pre.328 • channel master • https://github.com/flutter/flutter.git
Framework • revision da88baffd7 (61 minutes ago) • 2022-01-28 16:40:12 -0800
Engine • revision 2d3663cc4d
Tools • Dart 2.17.0 (build 2.17.0-67.0.dev) • DevTools 2.10.0-dev.1

/C:/Users/SKS%20Market/AppData/Local/Pub/Cache/hosted/pub.dartlang.org/bitsdojo_window_linux-0.1.1/lib/src/window.dart(145,19): error GCE1592F9: 'Size' is imported from both 'dart:ffi' and 'dart:ui'. [D:\_sksmarket\sksmarket\build\windows\flutter\flutter_assemble.vcxproj]
/C:/Users/SKS%20Market/AppData/Local/Pub/Cache/hosted/pub.dartlang.org/bitsdojo_window_linux-0.1.1/lib/src/window.dart(153,12): error GCE1592F9: 'Size' is imported from both 'dart:ffi' and 'dart:ui'. [D:\_sksmarket\sksmarket\build\windows\flutter\flutter_assemble.vcxproj]

**Comment by leech001** [comment]
#127 fixed double import of classes from 'dart:ffi' and 'dart:ui'

**Comment by selvam920** [comment]
without merge your code, how to use in pubspec.yaml file

**Comment by selvam920** [comment]
i'm getting below error

bitsdojo_window:
    git:
      url: https://github.com/leech001/bitsdojo_window
      ref: master
      path: bitsdojo_window 

Error on line 26, column 11: Invalid description in the "bitsdojo_window" pubspec on the "bitsdojo_window_platform_interface" dependency: "../bitsdojo_window_platform_interface" is a relative path, but this isn't a local pubspec.
   ╷
26 │     path: ../bitsdojo_window_platform_interface

**Comment by leech001** [comment]
change on you pubspec local path to package
```
bitsdojo_window:
    path: <path>/bitsdojo_window/bitsdojo_window/
```
local copy packages
[bitsdojo_window.zip](https://github.com/bitsdojo/bitsdojo_window/files/8007969/bitsdojo_window.zip)

**Comment by jonahzheng** [comment]
It seem fixing on flutter 2.10 .

**Comment by KnucklesEQ** [comment]
The error occurred in the master channel, but now also happens in the beta channel. 

We experienced it when we upgraded Flutter to 2.11.0-0.1.pre in beta channel. In stable channel still works fine... but I'm afraid that the next stable version of Flutter will bring this incompatibility to Bitsdojo...

**Comment by CodeDoctorDE** [comment]
It seems bitsdojo isn't active (see profile).

I have a small workaround with the path of lee001:

```yml
dependency_overrides: 
  bitsdojo_window_platform_interface:
    git:
      url: https://github.com/leech001/bitsdojo_window.git
      path: bitsdojo_window_platform_interface
  bitsdojo_window_windows:
    git:
      url: https://github.com/leech001/bitsdojo_window.git
      path: bitsdojo_window_windows
  bitsdojo_window_linux:
    git:
      url: https://github.com/leech001/bitsdojo_window.git
      path: bitsdojo_window_linux
  bitsdojo_window_macos:
    git:
      url: https://github.com/leech001/bitsdojo_window.git
      path: bitsdojo_window_macos
```

**Comment by bitsdojo** [comment]
Fixed in `bitsdojo_window: ^0.1.2` - https://github.com/bitsdojo/bitsdojo_window/commit/f320b623db107b03a5976fd88e52ec4fe81fa0c9

Closing this issue. Please re-open if needed.
